src/agents/ipp_group.jl

```diff
@@ -5077,6 +5101,13 @@ function solve_agent_problem_ipp_cap(
                     customers.x_DG_new_my(Symbol(Int(y_symbol)), h, z, m) for y_symbol in
                     model_data.year(first(model_data.index_y_fix)):model_data.year(y)
                 ) for h in model_data.index_h, m in customers.index_m
+            ) - 
+            # remove aggregated behind-the-meter pv/storage generation/consumption since they're front-of-the-meter now
+            sum(
+                customers.rho_DG(h, :BTMStorage, z, d, t) * der_aggregator.aggregation_level(reg_year_index_dera, z) * total_der_stor_capacity(z, h) for h in model_data.index_h
+            ) - 
+            sum(
+                customers.rho_DG(h, :BTMPV, z, d, t) * der_aggregator.aggregation_level(reg_year_index_dera, z) * total_der_pv_capacity(z, h) for h in model_data.index_h
```

Comment:
Is this removing PV and storage from the load profiles? Could that be done elsewhere, since it seems like someone (e.g., an LSE, but without an explicit LSE agent, maybe just the Customers module) should be passing the load profile (already processed) to the ISO?

Reply:
Elaine to take a look at streamlining information pass from customers to IPP

Reply:
It didn't make sense to calculate the full profiles in customers (since that would unnecessarily increase the data size, only to have it shrunk again later), but I was able to streamline the der-related code by calculating cumulative der capacity in customers.